src/BloomExe/ImageProcessing/ImageServer.cs

```diff
@@ -47,11 +47,13 @@ protected override void Dispose(bool fDisposing)
 			base.Dispose(fDisposing);
 		}
 
-		protected override bool StartWithSetupIfNeeded(out Exception error)
+		protected override void StartWithSetupIfNeeded()
 		{
-			var didStart = base.StartWithSetupIfNeeded(out error);
-
-			if(!didStart)
+			try
+			{
+				base.StartWithSetupIfNeeded();
+			}
+			catch (Exception error)
```

Comment:
Originally the code was assuming there was an exception if ` base.StartWithSetupIfNeeded(out error)` didn't return true. I simplified it to assume everything is OK if there isn't and exception.